src/dbus/constants.hpp

```diff
@@ -34,4 +34,35 @@
 #define DBUS_PROPERTY_GET_ALL_METHOD "GetAll"
 #define DBUS_PROPERTIES_CHANGED_SIGNAL "PropertiesChanged"
 
+#define OTBR_DBUS_PREFIX "io.openthread.BorderRouter"
+#define OTBR_DBUS_OBJECT_PREFIX "/io/openthread/"
+
+#define OTBR_DBUS_SCAN_METHOD "Scan"
+#define OTBR_DBUS_ATTACH_METHOD "Attach"
+#define OTBR_DBUS_FACTORY_RESET_METHOD "FactoryReset"
+#define OTBR_DBUS_RESET_METHOD "Reset"
+#define OTBR_DBUS_ADD_ON_MESH_PREFIX_METHOD "AddOnMeshPrefix"
+#define OTBR_DBUS_REMOVE_ON_MESH_PREFIX_METHOD "RemoveOnMeshPrefix"
+#define OTBR_DBUS_OPEN_UNSECURE_PORT_METHOD "OpenUnsecurePort"
+#define OTBR_DBUS_JOINER_START_METHOD "JoinerStart"
+#define OTBR_DBUS_JOINER_STOP_METHOD "JoinerStop"
+#define OTBR_DBUS_MESH_LOCAL_PREFIX_PROPERTY "MeshLocalPrefix"
+#define OTBR_DBUS_LEGACY_ULA_PREFIX_PROPERTY "LegacyULAPrefix"
+#define OTBR_DBUS_LINK_MODE_PROPERTY "LinkMode"
+#define OTBR_DBUS_DEVICE_ROLE_PROPERTY "DeviceRole"
+#define OTBR_DBUS_NETWORK_NAME_PROPERTY "NetworkName"
+#define OTBR_DBUS_PANID_PROPERTY "PanId"
+#define OTBR_DBUS_EXTPANID_PROPERTY "ExtPanId"
+#define OTBR_DBUS_CHANNEL_PROPERTY "Channel"
+#define OTBR_DBUS_MASTER_KEY_PROPERTY "MasterKey"
+#define OTBR_DBUS_CCA_FAILURE_RATE_PROPERTY "CcaFailureRate"
+#define OTBR_DBUS_LINK_COUNTERS_PROPERTY "LinkCounters"
+#define OTBR_DBUS_IP6_COUNTERS_PROPERTY "Ip6Counters"
+#define OTBR_DBUS_SUPPORTED_CHANNEL_MASK_PROPERTY "LinkSupportedChannelMask"
+#define DISABLED_ROLE_NAME "disabled"
```

Comment:
```suggestion
#define OTBR_DBUS_ROLE_DISABLED "disabled"
```
Also for the next 4 definitions.